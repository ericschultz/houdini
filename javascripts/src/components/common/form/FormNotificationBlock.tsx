// License: LGPL-3.0-or-later
import * as React from 'react';

class FormNotificationBlock extends React.Component<{}, {}> {
  render() {
     return <div className="help-block" role="alert">{this.props.children}</div>;
  }
}

export default FormNotificationBlock



