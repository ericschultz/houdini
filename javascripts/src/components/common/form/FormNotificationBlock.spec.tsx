// License: LGPL-3.0-or-later
import * as React from 'react';
import 'jest';
import FormNotificationBlock from './FormNotificationBlock'
import  toJson  from 'enzyme-to-json';
import { mount, shallow } from 'enzyme';

describe('FormNotificationBlock', () => {
  it('displays correctly', () => {
    let block = shallow(<FormNotificationBlock>a test </FormNotificationBlock>)

    expect(toJson(block)).toMatchSnapshot()
  })
})