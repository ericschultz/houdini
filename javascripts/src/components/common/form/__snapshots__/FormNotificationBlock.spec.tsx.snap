// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormNotificationBlock displays correctly 1`] = `
<div
  className="help-block"
  role="alert"
>
  a test 
</div>
`;
